code/game/machinery/_machines_base/machinery.dm

```diff
@@ -91,6 +91,7 @@ Class Procs:
 	var/malf_upgraded = 0
 	var/datum/wires/wires //wire datum, if any. If you place a type path, it will be autoinitialized.
 	var/use_power = POWER_USE_IDLE
+	var/silicon_restriction = FALSE // FALSE or one of the STATUS_* flags. If set, will force the given status flag if a silicon tries to access the machine.
```

Comment:
Kind of odd placement, in the middle of several power based vars

Reply:
Moved to the end of the vars list. Not sure why I put it there tbh.